azure_img_utils/azure_image.py

```diff
@@ -323,6 +324,45 @@ def add_image_to_offer(
             offer_doc
         )
 
+    def remove_image_from_offer(
+        self,
+        image_version: str,
+        offer_id: str,
+        publisher_id: str,
+        sku: str,
+        generation_id: str = None,
+        vm_images_key: str = None
+    ):
+        """
+        Delete the given image version from the offer.
+
+        The offer is pulled from the partner center, the old image version
+        is deleted and re-uploaded. To make the new image available
+        the offer must be published and set to go-live.
+
+        A blob_url is generated for the container if one is not provided.
+        """
+        offer_doc = self.get_offer_doc(offer_id, publisher_id)
+
+        kwargs = {
+            'generation_id': generation_id
+        }
+
+        if vm_images_key:
+            kwargs['vm_images_key'] = vm_images_key
+
+        offer_doc = remove_image_from_offer(
```

Comment:
Okay, this is why I don't like Python.

Reply:
Please fix this. By the time the reader is here the reader will have forgotten about the import on line 53. Instead of renaming you can change the imports and then use a namespace qualified call here if you want.